### DISCUSSION TITLE
Two remarks on the angle smoothing

### DISCUSSION THREAD

**Discussion by stla** [CGAL/cgal#7487] [General]
Hello,

I tried the angle and area smoothing today, with `.area_smoothing(false)`. 

I firstly tried with an EPECK mesh, and I had to kill the session so much it was slow. Maybe this should be mentionned in the doc: don't smooth an EPECK mesh.

My other remark is that the possibility to run the `PMP::angle_and_area_smoothing` without a `FaceRange` argument is not documented (but this is the example given).

**Comment by afabri** [comment]
Thank you for your feedback. The second remark is addressed in PR #7489.   

Concerning the first remark it would be a huge effort to do that systematically.
